rholang/src/main/scala/coop/rchain/rholang/interpreter/util/AddressTools.scala

```diff
@@ -0,0 +1,63 @@
+package coop.rchain.rholang.interpreter.util
+
+import java.nio.charset.StandardCharsets
+
+import coop.rchain.crypto.hash.Blake2b256
+import coop.rchain.rholang.interpreter.util.codec.Base58
+
+sealed case class Address(val prefix : String, val payload : Array[Byte])
+
+class AddressTools(val prefix : String) {
```

Comment:
Some methods in this class miss a return type.

Reply:
will fix